test/integration/master_routes_test.go

```diff
@@ -120,10 +120,6 @@ var expectedIndex = []string{
 	"/metrics",
 	"/oapi",
 	"/oapi/v1",
-	"/swagger-2.0.0.json",
```

Comment:
@liggitt you've added these 4 in https://github.com/soltysh/origin/commit/a84e38afedb#diff-5a30e93e2431231e8fd3d64ba241efeeR123 and that is causing this test to fail, any particular reason we need them?

Reply:
I just copied in what the server actually returned when I hit the root index. If the test returns something different, that's slightly concerning

Reply:
I've dropped `masterConfig.DisableOpenAPI = true` from `test/util/server/server.go` and reverted those 4 endpoints back. If the integration won't blow up we'll leave it as is, if not we'll handle this post-rebase.